new-dti-website/components/products/FloatingImages.tsx

```diff
@@ -56,8 +59,8 @@ export default function FloatingImages({ images }: FloatingImagesProps) {
           <div
             key={index}
             className={`${index === 0 ? 'relative z-10' : 'absolute'} w-full h-auto ${
-              index !== 0 ? getFloatClass(img.direction) : ''
-            } ${img.delay && index !== 0 ? 'floating-delay' : ''}`}
+              index !== 0 && !reduceMotion ? getFloatClass(img.direction) : ''
+            } ${!reduceMotion && img.delay && index !== 0 ? 'floating-delay' : ''}`}
```

Comment:
hmm for me it seems like the images still hover slightly even when with Reduce Motion settings on?

the lines don't animate which is expected but i still see the images move around a bit

https://github.com/user-attachments/assets/a4d7206d-d135-4d1c-8a20-1379b6549889

Reply:
Will look into this! Thanks for catching that!

Reply:
sure! lmk when u'd like me to re-review :)